src/api/app/models/user.rb

```diff
@@ -7,6 +7,7 @@ class User < ApplicationRecord
   # Keep in sync with states defined in db/schema.rb
   STATES = ['unconfirmed', 'confirmed', 'locked', 'deleted', 'subaccount'].freeze
   NOBODY_LOGIN = '_nobody_'.freeze
+  MAX_BIOGRAPHY_LENGTH_ALLOWED = 200
```

Comment:
:thought_balloon:  If you limit me like this you need to A) tell me and B) help me in the interface to keep the limit

Reply:
200 is also a *very* harsh limit for markdown formatted text

Reply:
true!

Reply:
this could be a start https://www.w3schools.com/tags/att_input_maxlength.asp

Reply:
> 200 is also a _very_ harsh limit for markdown formatted text

Is 250 or 255 enough for you? :innocent: :bouquet: I understand your point, but we used `string` type for the database field. In case we decide to extend it, we'll need another migration.

Reply:
As discussed offline, we are going to keep `biography`as string for now. We can extend it in the near future.